OWNERS

```diff
@@ -3,8 +3,6 @@
 approvers:
```

Comment:
Can you please send the OWNERS file change as part of separate PR. I think it is causing the tests to skip.

Reply:
https://github.com/openshift-helm-charts/development/pull/169 is raised